spec/jobs/create_feedback_session_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe CreateFeedbackSessionJob, type: :job do
+  describe '#perform' do
+    let(:provider_user) { create(:user) }
+    let(:receiver_user) { create(:user) }
+    let(:feedback_session_date) { Time.now.strftime("%Y-%m-%d") }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Rails/TimeZone: Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.